api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -382,17 +371,6 @@ public User apply(User user) {
     return userDao.findUsers();
   }
 
-  public User setIdVerificationApproved(Long userId, boolean blockscoreVerificationIsValid) {
```

Comment:
This method also appears to be unused: `getNonVerifiedUsers()`

Probably also the dao method that gets called there